### DISCUSSION TITLE
How do i fix my audio not working on my github pages website

### DISCUSSION THREAD

**Discussion by blgoreuxe** [blueedgetechno/win11React#274] [Q&A] [answered]
I am making a game in JavaScript and when you press an arrow key it should make it make a buzz sound if you pressed the wrong arrow or a slap sound if you pressed the correct one. The sounds do not play, although the background music does play still. the project was imported from replit, i have tried googling but i could not find a proper answer, i was also using a javascript plugin called howl. Howl is used to play audio in js.

Here is the github repo: [Github repo](https://github.com/blgoreuxe/ArrowBeat)

Here is the game on github: [game on github](https://blgoreuxe.github.io/ArrowBeat/)

Here is the game with a working slap and buzz sounds (on repl): [replit game](https://beat.brandongore502.repl.co/)

i want it to where the sounds work on the github link

**Comment by luni-moon** [comment, marked as answer]
This is something you ask on StackOverflow, not here.

**Reply by blgoreuxe** [reply]
i did but nobodys answering lol
but i fixed it anyways

**Reply by luni-moon** [reply]
> i did but nobodys answering lol
> 
> but i fixed it anyways

It's called *patience*...
